site/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "hexo-site",
+  "version": "0.0.0",
```

Comment:
We probably want to change these to `@embark/site` etc

Reply:
For sure 👍

Reply:
Even though it's a private package, maybe it's best to not use a namespace we don't own. Maybe `embark-site` instead?

Reply:
Yeap!